src/run.ts

```diff
@@ -0,0 +1,69 @@
+import { Helper } from '@dojo/interfaces/cli';
+import * as fs from 'fs-extra';
```

Comment:
does this need to be `fs-extra` anymore?

Reply:
Ah yeah the reason is I had some problems with mockery and the native `fs` module, I also saw we did it in another test which is where I got the idea